specification/communication/data-plane/PhoneNumbers/preview/2023-10-01-preview/phonenumbers.json

```
@@ -427,6 +427,59 @@
         }
       }
     },
+    "/availablePhoneNumbers/countries/{countryCode}/:browse": {
```

`POST` actions should not use a slash:

```suggestion
    "/availablePhoneNumbers/countries/{countryCode}:browse": {
```

If you already have `POST` actions that do this, however, feel free to ignore. You should be consistent within an API spec.

---

Our existing API already has the `/:` format for existing POST actions.